acestep/audio_utils.py

```
@@ -216,6 +216,10 @@ def save_audio(
             Actual saved file path
         """
         format = (format or self.default_format).lower()
+
+        force_flac = os.getenv("ACESTEP_FORCE_FLAC_EXPORT", "0") == "1"
+        if force_flac:
+            format = "flac" 
```

Ruff (0.15.9)

[error] Variable `format` is shadowing a Python builtin

(A001)